chain-crypto/src/algorithms/vrf/vrf.rs

```diff
@@ -244,20 +239,20 @@ impl OutputSeed {
     pub fn to_output(&self, input: &[u8], suffix: &[u8]) -> Blake2b256 {
         let mut buf = Vec::new();
         buf.extend_from_slice(input);
-        buf.extend_from_slice(self.0.compress().as_bytes());
+        buf.extend_from_slice(&self.0.to_bytes());
         buf.extend_from_slice(suffix);
 
         Blake2b256::new(&buf)
     }
 }
 
-fn make_message_hash_point(data: &[u8]) -> Point {
+fn make_message_hash_point(data: &[u8]) -> GroupElement {
     let m_hash = {
         let mut c = Sha512::new();
```

Comment:
`from_hash` already hashes data in input. I think it's redundant to hash it with `Sha512` before (and also a bit odd since we use blake everywhere else)